composer.json

```diff
@@ -17,7 +17,7 @@
   },
   "require-dev": {
     "laminas/laminas-cache": "^2.10",
-    "laminas/laminas-cache-storage-adapter-test": "^1.0.2",
+    "laminas/laminas-cache-storage-adapter-test": "1.0.2",
```

Comment:
Locking 1.1.x to 1.0.2 as 1.1.0 has BC breaks due to the version bump of phpunit...